### ISSUE TITLE
When I try to compile Arduino code, I get the following "error: arctos_moveit/ArmJointState.h: No such file or directory".  

### ISSUE THREAD

**Issue by Partap1234** [ArctosRobotics/ROS#1]

**Comment by Partap1234** [comment]
I tried this: 
cd <Arduino sketchbook>/libraries
rm -rf ros_lib 
rosrun rosserial_arduino make_libraries.py .
It made no difference

**Comment by iplayfast** [comment]
I ran into this myself. Apparently that file is generated by ros. Look https://github.com/jesseweisberg/moveo_ros/tree/master to see more detailed info.(I'm exploring myself so can't help much)

**Comment by Partap1234** [comment]
Yeah , I spent over 10 hours trying to figure it out. The file is in the build folder but Arduino cant seem to find it. The deeper I go the more problems I find. The URDF is missing all kinds of things. At this point I am just writing the code myself from scratch.

**Comment by Tenveis** [comment]
hello pratap, you need to source the **moveo_ros** workspace before building library for arduino. 
Here is the link of  updated readme: [Tenveis:arduino_compile](https://github.com/Tenveis/moveo_ros/tree/arduino_compile)
You can find the instructions in **Troubleshoot** section.
Hope this helps.
